modules/bitgo/src/v2/internal/tssUtils.ts

```diff
@@ -116,14 +116,15 @@ export class TssUtils extends MpcUtils {
    * @param backupKeyShare - backup's TSS key share
    * @param bitgoKeychain - previously created BitGo keychain; must be compatible with user and backup key shares
    * @param passphrase - wallet passphrase used to encrypt user's signing materials
+   * @param [originalPasscodeEncryptionCode]
    */
   async createUserKeychain(
     userGpgKey: openpgp.SerializedKeyPair<string>,
     userKeyShare: KeyShare,
     backupKeyShare: KeyShare,
     bitgoKeychain: Keychain,
     passphrase: string,
-    originalPasscodeEncryptionCode?: string
```

Comment:
Not sure if I should make this mandatory right away? Incase it's not passed in for some reason, wouldn't want it to block wallet creation. Thought we can do it after April 11.

Reply:
Making this mandatory to be consistent with api's requirement for TSS user key.